test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -733,6 +733,19 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
         $result.Output.RelationLink["self"] | Should -BeExactly "${baseUri}?maxlinks=5&linknumber=2&type=multiple"
     }
 
+
+    It "Validate Invoke-WebRequest RelationLink keys are treated case-insensitively" {
+        $uri = Get-WebListenerUrl -Test 'Link' -Query @{maxlinks = 5; linknumber = 2; type = 'multiple'}
+        $command = "Invoke-WebRequest -Uri '$uri'"
+        $result = ExecuteWebCommand -command $command
+
+        $result.Output.RelationLink["next"] -eq $result.Output.RelationLink["Next"] | Should -BeTrue
```

Comment:
Please redo these tests to be like the following:

```powershell
$result.Output.RelationLink["next"] | Should -Not -BeNullOrEmpty
$result.Output.RelationLink["next"] | Should -BeExactly $result.Output.RelationLink["Next"]
```

Repeat that pattern for the other assertions in this block.

This way we ensure that we are not somehow comparing `$null` to `$null` and getting a false pass.